samples/Gallery/Pages/ColorsByChannel.fs

```diff
@@ -0,0 +1,169 @@
+﻿namespace Gallery
+
+open System
+open System.Diagnostics
+open System.Reflection
+open Avalonia.Layout
+open Avalonia.Media
+open Fabulous
+open Fabulous.Avalonia
+
+(* Are these builders worth including into Fabulous.Avalonia in one form or the other?
```

Comment:
Yeah I thought about this and as this is not very common. It should ok to add empty widget constructor on demand like you have done here.